scripts/configure_petsc.sh

```diff
@@ -69,9 +69,11 @@ function configure_petsc()
 
   # If HDF5 is not found locally, download it via PETSc
   HDF5_FORTRAN_STR=''
+  HDF5_CONFIGURE_STR=''
   if [ -z "$HDF5_STR" ]; then
     HDF5_STR='--download-hdf5=1'
     HDF5_FORTRAN_STR='--download-hdf5-fortran-bindings=0'
+    HDF5_CONFIGURE_STR='--with-zlib'
```

Comment:
```suggestion
    HDF5_CONFIGURE_STR='--download-hdf5-configure-arguments="--with-zlib"'
```
This must have been a mistake in my commit

Reply:
Will fixup the original commit.